### DISCUSSION TITLE
No next page

### DISCUSSION THREAD

**Discussion by sanderlv** [HASwitchPlate/openHASP#399] [Q&A] [answered]
### Perform all steps below and tick them with [x]
- [x] Check the related part of the [Documentation](https://haswitchplate.github.io/openHASP-docs/)
- [x] Update openHASP to the latest version
- [x] Reproduce the issue and describe all steps

### Describe the bug
I have uploaded:
https://www.openhasp.com/0.6.3/integrations/openhab/integration_openhab/

I can see mqtt updates on touch.

But i can only get to the main screen, the buttons left and right do nothing. I only see the main screen.

### To Reproduce
Provide a small, independent code sample that can be used to reproduce the issue.
Format the code like this:
(https://www.openhasp.com/0.6.3/integrations/openhab/integration_openhab/)

### Expected behavior
I expect to see page 2 or 3

**Comment by fvanroie** [comment, marked as answer]
Use the `action` property and set it to either `prev`, `back` or `next` as described in this [pagination example](https://www.openhasp.com/0.6.3/integrations/examples/example-pagination/).

**Comment by sanderlv** [comment]
I this config:
https://www.openhasp.com/0.6.3/integrations/openhab/integration_openhab/

If I add this extra:
{"page":0,"id":1,"obj":"btn","action":"prev","x":0,"y":290,"w":79,"h":32,"bg_color":"#2C3E50","text":"\uE141","text_color":"#FFFFFF","radius":0,"border_side":0,"text_font":28}
{"page":0,"id":2,"obj":"btn","action":"back","x":80,"y":290,"w":80,"h":32,"bg_color":"#2C3E50","text":"\uE2DC","text_color":"#FFFFFF","radius":0,"border_side":0,"text_font":22}
{"page":0,"id":3,"obj":"btn","action":"next","x":161,"y":290,"w":79,"h":32,"bg_color":"#2C3E50","text":"\uE142","text_color":"#FFFFFF","radius":0,"border_side":0,"text_font":28}

It overwrites the original buttons. I would expect the config to be "complete"

**Comment by sanderlv** [comment]
Can I combine these? and how?

{"page":0,"comment":"---------- All pages ----------"}
{"obj":"btn","id":1,"x":5,"y":257,"w":73,"h":58,"bg_color":"#000000","border_color":"#C7BAA7","border_width":0,"radius":10,"radius10":10,"radius20":10,"text":"\uE04D","text_color":"#978B7D","text_font":32}
{"obj":"btn","id":2,"x":83,"y":257,"w":73,"h":58,"bg_color":"#000000","border_color":"#C7BAA7","border_width":0,"radius":10,"radius10":10,"radius20":10,"text":"\uE2DC","text_color":"#978B7D","text_font":32}
{"obj":"btn","id":3,"x":161,"y":257,"w":73,"h":58,"bg_color":"#000000","border_color":"#C7BAA7","border_width":0,"radius":10,"radius10":10,"radius20":10,"text":"\uE054","text_color":"#978B7D","text_font":32}

{"page":0,"id":1,"obj":"btn","action":"prev","x":0,"y":290,"w":79,"h":32,"bg_color":"#2C3E50","text":"\uE141","text_color":"#FFFFFF","radius":0,"border_side":0,"text_font":28}
{"page":0,"id":2,"obj":"btn","action":"back","x":80,"y":290,"w":80,"h":32,"bg_color":"#2C3E50","text":"\uE2DC","text_color":"#FFFFFF","radius":0,"border_side":0,"text_font":22}
{"page":0,"id":3,"obj":"btn","action":"next","x":161,"y":290,"w":79,"h":32,"bg_color":"#2C3E50","text":"\uE142","text_color":"#FFFFFF","radius":0,"border_side":0,"text_font":28}

Think I got it:
{"obj":"btn","action":"prev","id":1,"x":5,"y":257,"w":73,"h":58,"bg_color":"#000000","border_color":"#C7BAA7","border_width":0,"radius":10,"radius10":10,"radius20":10,"text":"\uE04D","text_color":"#978B7D","text_font":32}
{"obj":"btn","action":"back","id":2,"x":83,"y":257,"w":73,"h":58,"bg_color":"#000000","border_color":"#C7BAA7","border_width":0,"radius":10,"radius10":10,"radius20":10,"text":"\uE2DC","text_color":"#978B7D","text_font":32}
{"obj":"btn","action":"next","id":3,"x":161,"y":257,"w":73,"h":58,"bg_color":"#000000","border_color":"#C7BAA7","border_width":0,"radius":10,"radius10":10,"radius20":10,"text":"\uE054","text_color":"#978B7D","text_font":32}

Might be best to add that to the example?!!?
